week4_HW/mytodo-app/src/stores/index.js

```diff
@@ -0,0 +1,13 @@
+import DifferentValues from './DifferentValues'
+import TaskList from './TaskList'
+import { configure } from 'mobx'
+
+
+configure({ enforceActions: true })
+
+export default  {
+
+    values: DifferentValues,
+    data: TaskList
```

Comment:
These `values` and `data` names are not appropriate names for stores. All stores presumably have values and data in them. You should name your stores based on the category of data they are storing.